doc/how-to/replication/repl_bootstrap.rst

```diff
@@ -1,386 +1,659 @@
+
+    .. code-block:: yaml
+
+        replicaset001:
+          leader: instance002
+
+2.  Reload configuration on all instances using :ref:`config:reload() <config-module>`.
+
+3.  Make sure that ``instance002`` is a new master:
+
+    .. code-block:: console
+
+        manual_leader:instance002> box.info.ro
+        ---
+        - false
+        ...
+
+4.  Check replication status using ``box.info.replication``.
+
+
+..  _replication-remove_instances:
+
+Removing instances
+------------------
+
+This section describes the process of removing an instance from a replica set.
+
+Before removing an instance, make sure it is in read-only mode.
+If the instance is a master, perform :ref:`manual failover <replication-controlled_failover>`.
+
+..  _replication-remove_instances-disconnect:
+
+Disconnecting an instance
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1.  Clear the ``iproto`` option for ``instance003`` by setting its value to ``{}``:
+
+    .. code-block:: yaml
+
+        instance003:
+          iproto: {}
+
+2.  Reload configurations on ``instance001`` and ``instance002``:
+
+    -   ``instance001``:
+
+        .. code-block:: console
+
+            manual_leader:instance001> require('config'):reload()
+            ---
+            ...
+
+    -   ``instance002``:
+
+        .. code-block:: console
+
+            manual_leader:instance002> require('config'):reload()
+            ---
+            ...
+
+3.  Check that the ``upstream`` section is missing for ``instance003`` by executing ``box.info.replication[3]``:
+
+    .. code-block:: console
+
+        manual_leader:instance001> box.info.replication[3]
+        ---
+        - id: 3
+          uuid: 9a3a1b9b-8a18-baf6-00b3-a6e5e11fd8b6
+          lsn: 0
+          downstream:
+            status: follow
+            idle: 0.4588760000006
+            vclock: {1: 21}
+            lag: 0
+          name: instance003
+        ...
+
+
+..  _replication-remove_instances-stop:
+
+Stopping an instance
+~~~~~~~~~~~~~~~~~~~~
+
+1.  Stop ``instance003`` using the :ref:`tt stop <tt-stop>` command:
+
+    .. code-block:: console
+
+        $ tt stop manual_leader:instance003
+           • The Instance manual_leader:instance003 (PID = 15551) has been terminated.
+
+
+2.  Check that ``downstream.status`` is ``stopped`` for ``instance003``:
+
+    .. code-block:: console
+
+        manual_leader:instance001> box.info.replication[3]
+        ---
+        - id: 3
+          uuid: 9a3a1b9b-8a18-baf6-00b3-a6e5e11fd8b6
+          lsn: 0
+          downstream:
+            status: stopped
+            message: 'unexpected EOF when reading from socket, called on fd 27, aka 127.0.0.1:3301,
+              peer of 127.0.0.1:54185: Broken pipe'
+            system_message: Broken pipe
+          name: instance003
+        ...
+
+
+..  _replication-remove_instances-update-config:
+
+Removing an instance from the configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1.  Remove ``instance003`` from the ``instances.yml`` file:
```

Comment:
Minor: it would be natural to remove instance001, since we've already took leadership from it.